src/main/java/si/mazi/rescu/Config.java

```diff
@@ -104,6 +112,16 @@ private static boolean getBoolean(Properties properties, String key) {
         final String ignoreErrorCodes = properties.getProperty(key);
         return ignoreErrorCodes != null && Boolean.parseBoolean(ignoreErrorCodes);
     }
+    
+    private static Type getProxyTypeEnum(String proxyType) {
+      Type[] types = Type.values();
```

Comment:
Why not simply use ``Type.valueOf(proxyType)``?

Reply:
Will throw IllegalArgumentException if value is not SOCKS, HTTP or DIRECT but in fact, the error will be shown to the user, maybe this will be better ? I forgot to update the Readme accordly.

Reply:
Yes, I think an exception is better than returning null as it reveals the config error immediately.

Yes, please update the Readme.

You can cimply push the fixes into the same branch and it should show as part of this pull request.